docs/readme.md

```diff
@@ -4,17 +4,20 @@
 
 [![API status](https://badgen.net/uptime-robot/status/m793274556-25c5e9bbab0297d91cda7134)](https://stats.uptimerobot.com/57wNLs39M/793274556)
 
-Because it wraps [an API](https://github.com/public-transport/hafas-client/blob/6/readme.md#background) of [Deutsche Bahn](https://de.wikipedia.org/wiki/Deutsche_Bahn), it **includes most of the long-distance and regional traffic, as well as some international trains and local buses**. Essentially, it returns whatever data the [*DB Navigator* app](https://www.bahn.de/p/view/service/mobile/db-navigator.shtml) shows, **including realtime delays and disruptions**.
+Because it wraps [an API](https://github.com/public-transport/hafas-client/blob/6/readme.md#background) of [Deutsche Bahn](https://de.wikipedia.org/wiki/Deutsche_Bahn), it **includes most of the long-distance and regional traffic, as well as some international trains and local buses**. Essentially, it returns whatever data the [*DB Navigator* app](https://www.bahn.de/p/view/service/mobile/db-navigator.shtml) shows*, **including realtime delays and disruptions**.
+
+*When comparing realtime information for past journeys, sometimes differing results can be observed between the API response and what the DB Navigator app shows. This might be due to implementation details in the app.*
 
 - [Getting Started](getting-started.md)
-- [API documentation](api.md)
-- [OpenAPI playground](https://petstore.swagger.io/?url=https%3A%2F%2Fv6.db.transport.rest%2F.well-known%2Fservice-desc%0A)
+- [OpenAPI playground with API documentation](https://petstore.swagger.io/?url=https%3A%2F%2Fv6.db.transport.rest%2F.well-known%2Fservice-desc%0A)
 
 ## Why use this API?
 
 ### Realtime Data
 
-This API returns realtime data whenever its upstream, the [API for DB's mobile app](https://github.com/public-transport/hafas-client/blob/6/p/db/readme.md), provides it.
+This API returns realtime data whenever it is upstream. The [API for DB's mobile app](https://github.com/public-transport/hafas-client/blob/6/p/db/readme.md) provides it.
+
+*Note: Realtime data like delays and cancellations disappear from the provided data roughly 2hrs after the journey's arrival time.*
```

Comment:
Thats not true, there is no rule when stuff disappears, we just know that some endpoints (like journeyDetails or tripSearch) will remove real time data.
Others like SearchOnTrip (I'm not sure if this project supports it) will removev itlater but will still remove it.

Reply:
I see. How about this then:

> Note: Different endpoints might remove realtime data like delays and cancellations at different times, i.e. after a journey's arrival.